### ISSUE TITLE
Upgrade to python-irodsclient v2.2.0

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#2023]
**Note:** This version used to be called 2.1.1 but it was bumped to 2.2.0

This update fixes the forced file open redirect issue which has been causing problems (see #2007, #2022).

There is no due date, but at the time of writing the [milestone](https://github.com/irods/python-irodsclient/milestone/5) for this release is at 85% with 2 remaining issues, so should hopefully be fairly soon.

Adding as a separate ticket to #1959 as this is quite a relevant upgrade.

Once this is done, should remove references to /etc/hosts hacks from the documentation and also remove it from the CI workflow.

**Update:** No open issues left in the milestone now, still not tagged though..

**Comment by mikkonie** [comment]
The 2.2.0 release is now out. Go go go!

**Comment by mikkonie** [comment]
Done. This fixed the need for setting hostnames for localhost in dev/CI. Github Actions and docs have been updated accordingly.
